internal/config/config.go

```diff
@@ -241,12 +205,11 @@ func ParseConfig(rd io.Reader) (Config, error) {
 }
 
 type CombinedSettings struct {
-	Global    Config
-	Package   SQL
-	Go        SQLGo
-	JSON      SQLJSON
-	Rename    map[string]string
-	Overrides []Override
+	Global  Config
+	Package SQL
+	Go      golang.Options
+	JSON    SQLJSON
+	Rename  map[string]string
```

Comment:
I don't think this Rename map is used anymore? Because in `codegen()` we pull everything from `combo.Global.Overrides.Go`.